IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ImpactProtectionDeviceTypeBumper/DocPropertySet.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_ImpactProtectionDeviceTypeBumper" UniqueId="7377882c-5384-726c-ebb0-55ea34a83504" ApplicableType="IfcImpactProtectionDevice/IfcImpactProtectionDeviceTypeEnum(.BUMPER.)" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="BrakingLength_3S5C4lkGf3_vevKH2HI3L4" />
```

Comment:
Description of this property relates to a design value, usually a property set of this type and naming should describe the limits of the element not the design criteria applied I would generalise the description to not be specifically design criteria and if design parameters are needed then consider a _DesignCriteria_ sister Pset similar to that used for Fenders

Reply:
I created a new property set Pset_ImpactProtectionDeviceOccurrenceBumper, because there are other two properties should be defined in occurrence level. I think taking this property in occurrence level is OK. If create another one just for design criteria, then that one will only have one property, not very nice.